src/workloads/query/GroupStagesOnComputedFields.yml

```diff
@@ -0,0 +1,176 @@
+      pipeline: [{$addFields:
+                          {"hourDiff": {$dateDiff: {"startDate": "$time", "endDate": {^Date: "2022-01-04"}, "unit": "hour"}}}},
+                          {$match: {"hourDiff": {$lte: 12}}},
+                          {$group:
+                            {"_id" :
+                              { "symbol" : "$symbol" ,
+                                "time": { "$dateTrunc" : {"date": "$time", "unit": "minute", "binSize" : 60}}
+                              },
+                              "open" : {"$first" : "$price"},
+                              "close" : {"$last" : "$price"}}
+                          },
+                          {
+                              $addFields : {
+                                  "diffPercentage" : {$round : [{$multiply : [100, { $divide : [{$subtract : ["$close", "$open"]}, "$open"] }]}, 2]}
+                              }
+                          },
+                          {
+                            $group: {
+                                  "_id" : "$_id.time",
+                                  "topN": {
+                                      $topN: {
+                                          "output": ["$_id.symbol", "$diffPercentage", "$close"],
+                                          "sortBy" : { "diffPercentage" : -1 },
+                                          "n": 3
+                                      }
+                                  },
+                                  "bottomN": {
+                                      $bottomN: {
+                                          "output": ["$_id.symbol", "$diffPercentage", "$close"],
+                                          "sortBy" : { "diffPercentage" : 1 },
+                                          "n": 3
+                                      }
+                                  },
+                              }
+                          },
+                          {
+                              $sort : {
+                                  "_id" : -1
+                              }
+                          }]
+
+GroupStageAfterMatchOnDateComputation: &GroupStageAfterMatchOnDateComputation
+  LoadConfig:
+    <<: *loadConfig
+    Parameters:
+      name: GroupStageAfterMatchOnDateComputation
+      active: {^Parameter: {Name: "active", Default: [4]}}
+      nopInPhasesUpTo: *NopInPhasesUpTo
+      pipeline: [{$match: {
+                          "symbol": "string050",
+                          "time": {
+                            $gte: "2022-01-03"
+                          }
+                          }
+                  },
+                  {$group:
+                    {"_id" : {
+                        $dateTrunc: {
+                          "date": "$time",
+                          "unit": "day"
+                        }
+                      },
+                      "count" : {$sum: 1}
+                    }},
```

Comment:
The second bracket (closing the `$group`) should be on its own line, and the subsequent `$sort` should be indented one level less.

Reply:
Done.